### ISSUE TITLE
failed to load json config (NRZ-2021-134-B4 (22 Aug))

### ISSUE THREAD

**Issue by Phaze-III** [opendata-stuttgart/sensors-software#921]
Under certain circumstances NRZ-2021-134-B4 fails to parse an updated configuration file with a setting "Höhe über Meeresspiegel (m)" that has more than 1 digit as height above sealevel. So 0 to 9 will work, 10 or higher fails.
This appears to be related to disabling the auto-update feature.

Steps to reproduce:

1. erase flash and flash firmware.sensor.community/airrohr/beta/latest_de.bin
2. configure sensor with at least "Lade Beta Versionen" enabled to prevent a downgrade to release and debug level to 5 to get more debug info
3. under Sensoren set "Höhe über Meeresspiegel (m)" to 25 ( a two digit height)
4. save and restart
5. the sensor will reboot and have the correct height configured.
6. disable the Auto-Update completely
7. save and restart

The sensor will restart but the change in step 6. is gone (Auto-Update still enabled). In de debug output it shows:
```airRohr: NRZ-2021-134-B4/DE
mounting FS...
opened config file...
failed to load json config
opened config file...
parsed json...
output debug text to displays...
```
Basically the new config fails to load, falling back to the previous version.

The same steps using a height below 10 (1 digit) does work.

Note that writing a 'failed' config two times in a row results in no configuration at all (both current and previous fail). Should this be a separate issue?
